drake/solvers/test/testOptimizationProblem.cpp

```diff
@@ -192,10 +192,88 @@ void gloptipolyConstrainedMinimization() {
   valuecheckMatrix(x.value(), Vector3d(.5, 0, 3), 1e-4);
 }
 
+/**
+ * Test that the eval() method of LinearComplementarityConstraint correctly
+ * returns the slack.
+ */
+void simpleLCPConstraintEval() {
```

Comment:
I have ported testOptimizationProblem to googletest in another branch.  It will cause conflicts with this PR, which is why I have not pushed it.

I will open appropriate PRs once this one is finally merged and I can prepare a non-conflicting branch.